### DISCUSSION TITLE
v0.35.2

### DISCUSSION THREAD

**Discussion by rfratto** [grafana-cold-storage/agent#4625] [Announcements]
This is release `v0.35.2` of Grafana Agent.

### Upgrading

Read the relevant upgrade guides for specific instructions on upgrading from older versions:

* [Static mode upgrade guide](https://grafana.com/docs/agent/v0.35/static/upgrade-guide/)
* [Static mode Kubernetes operator upgrade guide](https://grafana.com/docs/agent/v0.35/operator/upgrade-guide/)
* [Flow mode upgrade guide](https://grafana.com/docs/agent/v0.35/flow/upgrade-guide/)

#### Bugfixes

- Fix issue where the flow mode UI would show an empty page when navigating to
  an unhealthy `prometheus.operator` component or a healthy
  `prometheus.operator` component which discovered no custom resources.
  (@rfratto)

- Fix panic when using `oauth2` without specifying `tls_config`. (@mattdurham)

- Fix issue where series records would never get written to the WAL if a scrape
  was rolled back, resulting in "dropped sample for series that was not
  explicitly dropped via relabelling" log messages. (@rfratto)

- Fix RPM file digests so that installation on FIPS-enabled systems succeeds. (@andrewimeson)

#### Other changes

- Compile journald support into builds of `grafana-agentctl` so
  `grafana-agentctl test-logs` functions as expected when testing tailing the
  systemd journal. (@rfratto)

### Installation

Refer to our installation guides for how to install the variants of Grafana Agent:

* [Install static mode](https://grafana.com/docs/agent/v0.35/static/set-up/install/)
* [Install the static mode Kubernetes operator](https://grafana.com/docs/agent/v0.35/operator/helm-getting-started/)
* [Install flow mode](https://grafana.com/docs/agent/v0.35/flow/setup/install/)

<hr /><em>This discussion was created from the release <a href='https://github.com/grafana/agent/releases/tag/v0.35.2'>v0.35.2</a>.</em>

**Comment by andrewimeson** [comment]
So that I know when to re-test my fix for (#4419) with the production release, how long does it take for the RPMs to be updated on rpm.grafana.com?

**Reply by mattdurham** [reply]
Its a nightly UTC job
